pkg/driver/centos.go

```diff
@@ -10,8 +11,15 @@ type CentOSDriver struct {
 	systemdUnitOverwriteDriver
 }
 
-func (c CentOSDriver) InstallDocker() error {
-	return executil.ExecPipe("/bin/sh", "-c", "curl -sSL https://get.docker.com/ | sh")
+func (c CentOSDriver) InstallDocker(azureEnv string) error {
+	switch azureEnv {
+		case "AzureCloud":
+			return executil.ExecPipe("/bin/sh", "-c", "curl -sSL https://get.docker.com/ | sh")
```

Comment:
@lizzha it looks like the only thing that's different is the last argument to `ExecPipe`. can you please remove duplication and just choose that one based on `azureEnv`?